proto/message_contents/invitation.proto

```diff
@@ -19,13 +19,19 @@ message InvitationV1 {
         bytes key_material = 1;  // randomly generated key material (32 bytes)
     }
 
+    message GroupContext {
+        repeated string initialMembers = 1;
+    }
+
     // The context type
     message Context {
         // Expected to be a URI (ie xmtp.org/convo1)
         string conversation_id = 1;
         // Key value map of additional metadata that would be exposed to
         // application developers and could be used for filtering
         map<string, string> metadata = 2;
+
+        optional GroupContext group_context = 3;
```

Comment:
Do we actually need the `optional` flag here? Objects are always optional in `proto`, and should default to an empty object when read with an unset value. The `optional` flag is really made for scalar fields, where it creates a wrapper object to differentiate between the default value and the value being unset.

The Typescript types do expect that you set a value for an object field, but we can work around that at the SDK layer and set our own defaults

Reply:
Gotcha, I'm not what you'd call a protobuf expert so I appreciate that context. I can remove the optional (and the flag passing in the go generation).